src/libraries/Microsoft.PowerFx.LanguageServerProtocol/LanguageServer/LanguageServer.cs

```diff
@@ -279,9 +282,85 @@ private void HandleInitialFixupRequest(string id, string paramsJson)
             }));
         }
 
+        private void HandleCodeActionRequest(string id, string paramsJson)
```

Comment:
>HandleCodeActionRequest [](http://example.com/codeflow?start=21&length=23)

What user gesture causes this to get called? 
(ie, how frequently does it get called...)

Reply:
It gets called only when error occurs (conditionally not always)